views/index.ejs

```diff
@@ -50,5 +53,6 @@
       <script>window.times = {init : Date.now()}</script>
     <% } %>
     <script src="./statics/build/javascript/bundle.js"></script>
+    <noscript><%= _('JavaScript is required to view this application.') %></noscript>
```

Comment:
this element is rendered behind the "top_bar" :crying_cat_face:

Reply:
I fixed it in the last commit ![localhost_3000_(Galaxy S5) (1)](https://user-images.githubusercontent.com/243653/63497893-14518900-c4c5-11e9-9f27-50131d03b4e2.png)